django/db/models/fields/related.py

```diff
@@ -1488,8 +1489,16 @@ def _get_m2m_db_table(self, opts):
         elif self.db_table:
             return self.db_table
         else:
-            m2m_table_name = '%s_%s' % (utils.strip_quotes(opts.db_table), self.name)
-            return utils.truncate_name(m2m_table_name, connection.ops.max_name_length())
+            # If db_table contains username e.g. "USERNAME"."TABLE" we pass to
```

Comment:
Write comments without "we", e.g. "."TABLE", pass only table name to truncate_name()."